docs/en/observability/apm/known-issues.asciidoc

```diff
@@ -34,9 +34,28 @@ Failed installing package [apm] due to error: [ResponseError: mapper_parsing_exc
 		mapper_parsing_exception: Field [jvm.memory.non_heap.pool.committed] attempted to shadow a time_series_metric]
 ----
 
-// Link to fix?
+// Link to fix
 A fix was released in 8.11.2: https://github.com/elastic/kibana/pull/171712[elastic/kibana#171712].
 
+[[apm-empty-metricset-values]]
+*APM Server accepts empty metricset values that should have been rejected* +
+_APM Server versions: < 8.11.0_ +
+_Elastic APM Java Agent versions: < 1.43.0_
+
+// Describe the conditions in which this issue occurs
+If you are using APM Server (< v8.11.0) and/or the Elastic APM Java Agent (< v1.43.0),
+// Describe the behavior of the issue
+the agent may be sending empty histogram metricsets.
+// Describe why it happens
+This issue was a result of validation not being properly applied.
```

Comment:
How about: 
> In previous APM Server versions some data validation was not properly applied, leading to the APM Server excepting empty histogram metricsets where it shouldn't. This bug was fixed in the APM Server in `8.11.0`. 
> The APM Java Agent (< v1.43.0) was sending such invalid data under certain circumstances. 
> Without upgrading the APM Java Agent version, this can lead to metricsets being rejected by the APM Server and additional error logs in the Java Agent.